.env.example

```diff
@@ -1,18 +0,0 @@
-# WEBAPP
```

Comment:
Why was `.env.example` removed?

Reply:
my mistake, in the documentation (https://docs.human-connection.org/documentation/web-app/installation) it mentions .env file so when i first installed the product i changed .env.example by .env